src/validataclass/helpers/unset_value.py

```diff
@@ -39,8 +39,8 @@ def __str__(self):
     def __bool__(self):
         return False
 
-    def __eq__(self, other):
-        return other is self
+    # Don't define __eq__ because the default implementation is fine (identity check), and because we would then have to
```

Comment:
Then why you don't remove the line?

Reply:
I'm not sure if I got your question ^^' You mean why I put the comment there instead of just removing the code?

Reply:
Mainly as a reminder for future me who might possibly see this class and say "hmm, why didn't we define `__eq__` here, maybe I should add that, that sounds like a good idea" :sweat_smile: